src/controllers/dacpacDialogWebviewController.ts

```diff
@@ -541,79 +554,35 @@ export class DacFxApplicationWebviewController extends ReactWebviewPanelControll
-                const displayName = this.buildConnectionDisplayName(profile);
-                const profileId = profile.id || `${profile.server}_${profile.database || ""}`;
-
-                // Check if this connection is active and properly connected
-                const ownerUri = this.connectionManager.getUriForConnection(profile);
-                const isConnected =
-                    ownerUri && activeConnections[ownerUri]
-                        ? this.connectionManager.isConnected(ownerUri)
-                        : false;
-
-                connections.push({
-                    displayName,
-                    server: profile.server,
-                    database: profile.database,
-                    authenticationType: this.getAuthenticationTypeString(
-                        profile.authenticationType,
-                    ),
-                    userName: profile.user,
-                    isConnected,
-                    profileId,
-                });
-            }
-
-            const existingProfileIds = new Set(connections.map((conn) => conn.profileId));
+                // Use getConnectionDisplayName if profileName is not set
+                const displayName = profile.profileName || getConnectionDisplayName(profile);
 
-            // Include active connections that may not appear in the recent list
-            for (const activeConnection of Object.values(activeConnections)) {
-                const profile = activeConnection.credentials as IConnectionProfile;
-                const profileId = profile.id || `${profile.server}_${profile.database || ""}`;
-
-                if (existingProfileIds.has(profileId)) {
-                    continue;
-                }
-
-                // Only include if actually connected (not in connecting state or errored)
-                const ownerUri = this.connectionManager.getUriForConnection(profile);
-                if (!ownerUri || !this.connectionManager.isConnected(ownerUri)) {
-                    continue;
-                }
-
-                const displayName = this.buildConnectionDisplayName(profile);
-
-                connections.push({
-                    displayName,
+                return {
                     server: profile.server,
                     database: profile.database,
-                    authenticationType: this.getAuthenticationTypeString(
-                        profile.authenticationType,
-                    ),
-                    userName: profile.user,
-                    isConnected: true,
-                    profileId,
-                });
-                existingProfileIds.add(profileId);
-            }
+                    user: profile.user,
+                    password: profile.password,
+                    authenticationType: profile.authenticationType,
+                    profileName: displayName,
+                    id: profile.id || `${profile.server}_${profile.database || ""}`,
```

Comment:
We should never have a profile without an ID.  Have you seen that here before?

If we create one, it should be a GUID, but we shouldn't need to do that.